media-gfx/alembic/files/alembic-1.8.6-imath.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/python/PyAlembic/Tests/testCollections.py b/python/PyAlembic/Tests/testCollections.py
```

Comment:
From what the patch does, I know this is a py3.12 ptch -- but there's no comment at the top saying where it's from or why or what... :)